lldb/source/Plugins/Language/Swift/LanguageSwiftProperties.td

```diff
@@ -0,0 +1,8 @@
+include "../../../../include/lldb/Core/PropertiesBase.td"
+
+let Definition = "language_swift" in {
```

Comment:
See my comments in https://github.com/swiftlang/llvm-project/pull/10710/files/bd5d5023865c6a2fc7e613468749165723241e92..6f22fc8a6d93c15796b5b869be1ac6614c76ed69

Reply:
FWIW, I think keeping it consistent with all the other plugin setting names seems like the right thing to do